gapic-generator-cloud/bin/protoc-gen-ruby_cloud

```diff
@@ -14,6 +14,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+$LOAD_PATH.unshift ::File.expand_path("../../lib", __FILE__)
```

Comment:
Better: `::File.expand_path("../lib", __dir__)`
(Actually, I prefer to see `::File.join(::File.dirname(__dir__), "lib")` for better cross-platform compatibility, but it probably doesn't matter that much for this file.)

Can you also keep a blank line after the copyright notice comments above?